packages/react-components/react-nav-preview/src/components/NavSubItemGroup/renderNavSubItemGroup.tsx

```diff
@@ -10,6 +10,5 @@ import type { NavSubItemGroupState, NavSubItemGroupSlots } from './NavSubItemGro
 export const renderNavSubItemGroup_unstable = (state: NavSubItemGroupState) => {
   assertSlots<NavSubItemGroupSlots>(state);
 
-  // TODO Add additional slots in the appropriate place
-  return <state.root />;
+  return state.open ? <state.root>{state.root.children}</state.root> : null;
```

Comment:
I think you can avoid this logic here and move it to the state. When resolving the slot you could make it optional since it doesn't always render and set it to undefined if not open.

Reply:
Part of the issue with the conformance tests is the one I mentioned above. The conformance test is not finding anything because the state is not open, therefore rendering just null. There are two ways to go about this:
* Make the component always render something: Ignore the comment above and instead make sure it always renders something (looking at the implementation it's not the solution you should go for).
* Adjust the conformance tests to make sure something is being rendered and the conformance tests can work correctly. In this case you could add a wrapper to the component with the context sending open=true.

Reply:
Closing - we figured out how to mock the appropriate values in the conformance tests.